app/build.gradle.kts

```diff
@@ -0,0 +1,69 @@
+plugins {
+    alias(libs.plugins.androidApplication)
+    alias(libs.plugins.jetbrainsKotlinAndroid)
+}
+
+android {
+    namespace = "edu.stanford.spezikt"
```

Comment:
We can do that for sure 👍 

However, I would suggest to update after the first PRs have been merged. The reason is that the namespace is also used in every import and folder structures in the whole project, e.g. `edu/stanford/spezikt/*`. There will be a lot of merge conflicts to do it now, and not a suggested task to be done manually. I have once done something similar with a swift CLI, and can do that in one shot - (thank you ApodiniMigrator 😂)

Reply:
Makes sense, let's transform this into an issue. I think it will always be a pain point to do this; probably wise to do it sooner than later after the first version of the Bluetooth PR is merged?

Reply:
yes, it makes sense to do this promptly afer first changes are merged, as everything else is also based on it

Reply:
Thank you @Basler182 & @eldcn 👍